### DISCUSSION TITLE
Changing the Infobox width

### DISCUSSION THREAD

**Discussion by WardUlenaers** [SlRvb/Obsidian--ITS-Theme#141] [Q&A] [answered]
I am using the latest ITS theme and love it, though I would like to change the width of the infobox.
I have l tried a lot of different things but can't seem to make it work.
Is there an option to do this that I am looking over or isn't this possible at the moment?

**Comment by ghost** [comment]
Having similar issues ... Would be a huge improvement to have the ability to change/set infobox width.

**Comment by Ji11ard** [comment]
I took a look in the css snippet itself, and it seems to implement the following size modifiers, you just append them after the `|` in the infobox definition. I've put the width percentages in brackets so you can see how big each one is. 

wtiny (20%)
wsmall (30%)
ws-med (40%)
wm-sm (50%)
wmed (60%)
wm-tl (80%)
wtall (95%)
wfull (100%)

wtiny-c (19%)
wsmall-c (32.4%)
ws-med-c (39%)
wm-sm-c (49%)
wmed-c (59%)
wm-tl-c (79%)

I think the "c" versions are complements if you want a left and right inbox side-by-side.

**Comment by jeremywhalen** [comment]
It's been a week but here is what I recall doing. I copied the /*Infobox*/ section out of the ITS theme.css and pasted it into a new css file. I then renamed it to infoboxnpc and changed max-width from 300px to 475px;) 

Here is my custom CSS file for reference.
[infoboxnpc.zip](https://github.com/SlRvb/Obsidian--ITS-Theme/files/11876858/infoboxnpc.zip)

**Comment by SlRvb** [comment, marked as answer]
Does this not work for yall? https://publish.obsidian.md/slrvb-docs/ITS+Theme/Callout+Adjustments#Sizing

**Reply by Ji11ard** [reply]
Works for me - I'd just been using the [old guide](https://github.com/SlRvb/Obsidian--ITS-Theme/blob/main/Guide/Callouts.md) when I was playing last year, saw that there was a 'wsmall' used in an example, and went trawling through the css to find the others 🙂

**Reply by jeremywhalen** [reply]
Using callout widths I can make the infobox narrower or wfull but otherwise the maxwidth setting (line 25 in the css file I shared earlier) seems to limit the width, which is logical. That is why I took the approach of creating a new callout type based entirely on the the ITS Infobox with the exception of a slightly wider max-width.
